history/src/postgres/postgres_block.rs

```diff
@@ -1,45 +1,83 @@
-use super::postgres_session::SchemaSize;
-use solana_lite_rpc_core::{
-    commitment_utils::Commitment, encoding::BASE64, structures::produced_block::ProducedBlock,
-};
+use solana_lite_rpc_core::{encoding::BASE64, structures::produced_block::ProducedBlock};
+use tokio_postgres::types::ToSql;
+
+use super::postgres_session::PostgresSession;
 
 #[derive(Debug)]
 pub struct PostgresBlock {
-    pub leader_id: Option<String>,
+    pub slot: i64,
     pub blockhash: String,
     pub block_height: i64,
-    pub slot: i64,
     pub parent_slot: i64,
     pub block_time: i64,
-    pub commitment_config: i8,
     pub previous_blockhash: String,
     pub rewards: Option<String>,
 }
 
-impl SchemaSize for PostgresBlock {
-    const DEFAULT_SIZE: usize = 4 * 8;
-    const MAX_SIZE: usize = Self::DEFAULT_SIZE + 8;
-}
+const NB_ARUMENTS: usize = 7;
 
-impl From<ProducedBlock> for PostgresBlock {
-    fn from(value: ProducedBlock) -> Self {
-        let commitment = Commitment::from(&value.commitment_config);
+impl From<&ProducedBlock> for PostgresBlock {
+    fn from(value: &ProducedBlock) -> Self {
         let rewards = value
             .rewards
             .as_ref()
             .map(|x| BASE64.serialize(x).ok())
```

Comment:
maybe we should add a version tag: I use this pattern: `v1#ZDM0ODcyNDUwMjViMjM0NGJhNDc4ZDgyYTVjODM5OGQ4Nzg1OWQ3N2IzZmE3NDAyMmNlYTAyYzljYjMwYjA1OAo=`